conda_forge_tick/git_utils.py

```diff
@@ -40,13 +42,52 @@
 
 CF_BOT_NAMES = {"regro-cf-autotick-bot", "conda-forge-linter"}
 
+PR_KEYS_TO_KEEP = {
+    "id": None,
+    "number": None,
+    "html_url": None,
+    "created_at": None,
+    "updated_at": None,
+    "merged_at": None,
+    "state": None,
+    "mergeable_state": None,
+    "labels": None,
+    "head": {"ref": None},
+    "base": {"repo": {"name": None}},
+}
```

Comment:
@CJ-Wright Can you look at the github api docs (https://docs.github.com/en/free-pro-team@latest/rest/reference/pulls#get-a-pull-request) and see what you think should be here?

Reply:
I think this is ok, I need to look at `by-the-numbers` to see if we need any more keys.